woocommerce/woocommerce-gateway-stripe
# Add E2E tests shopper checkout

Closes #2485 

## Changes proposed in this Pull Request:

This PR adds the following E2E test cases:

[Shopper - Checkout: Checkout with normal credit card](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Testing-instructions-for-critical-flows-using-the-classic-checkout-experience#checkout-with-normal-credit-card)
 [Shopper - Checkout: Checkout with saved card](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Testing-instructions-for-critical-flows-using-the-classic-checkout-experience#checkout-with-saved-card)
 [Shopper - Checkout: Checkout with SCA card](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Testing-instructions-for-critical-flows-using-the-classic-checkout-experience#checkout-with-sca-card)
 [Shopper - Checkout: Checkout failures](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Testing-instructions-for-critical-flows-using-the-classic-checkout-experience#checkout-failures-with-various-cards)

## Testing instructions

1. Follow the instructions [here](https://github.com/woocommerce/woocommerce-gateway-stripe/tree/develop/tests/e2e)  to setup a new E2E environment using a jurassic ninja site.
2. Run the E2E tests.

All tests should pass.

---


**Post merge**
